rzup/README.md

```diff
@@ -0,0 +1,41 @@
+# `rzup`
+
+Install, update, or revert to a specific RISC Zero version.
+
+## Install
+
+<!-- TODO: Replace this with the correct url path once merged. -->
+```sh
+curl -L https://raw.githubusercontent.com/risc0/risc0/hmrtn/rzup-installer/rzup/install | bash
```

Comment:
@morgangibbs87 There's an existing bucket we can use for this. DM for details.

Reply:
replaced with: https://risc0-artifacts.s3.us-west-2.amazonaws.com/rzup/install